terra-jupyter-aou/Dockerfile

```diff
@@ -45,6 +45,7 @@ ENV PYSPARK_PYTHON=python3
 
 RUN pip3 install pypandoc \
       nbstripout \
+      nbdime \
```

Comment:
I'm fine with this - @Qi77Qi @rtitle any interest in pushing this upstream to the python base?

Reply:
If there is interest, I'd be happy to update this PR to make the change there, just let me know!

Both `nbstripout` and `nbdime` should move together.